website/src/data/users.tsx

```diff
@@ -1535,6 +1535,14 @@ const Users: User[] = [
     source: 'https://github.com/whirl21/website',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: 'WoodpeckerCI',
+    description: "Woodpecker is a simple CI engine with great extensibility.",
+    preview: require('./showcase/woodpecker.png'),
+    website: 'https://woodpecker-ci.org/',
+    source: 'https://github.com/woodpecker-ci/woodpecker/tree/master/docs',
```

Comment:
```suggestion
    source: 'https://github.com/woodpecker-ci/woodpecker',
```

Reply:
No need for pointing to subdirectories :)

Reply:
@Josh-Cena should I squash the fix?

Reply:
No need, we squash merge anyways. Thanks!